Proiect-PAO/src/Services/IMainService.java

```diff
@@ -0,0 +1,46 @@
+package Services;
+
+import Client.Client;
+import Date.Date;
+import Event.*;
+import Location.*;
+import Date.*;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Scanner;
+
+public interface IMainService {
```

Comment:
you can split this into several services  with their implementations - one for client, one for location (LocationService for example), etc